backend/src/main/java/ai/verta/modeldb/utils/RdbmsUtils.java

```diff
@@ -1771,6 +1786,77 @@ public static Order getOrderBasedOnSortKey(
     return finalPredicatesList;
   }
 
+  private static void setFuzzyOwnerPredicateBasedOnUACWorkspace(
```

Comment:
I am confused. I'd assume this code already existed before and you're just moving, but it seems to have a lot of new logic added to it. Am I missing something here?

Reply:
https://github.com/VertaAI/modeldb/pull/1838/files#r554756537
this method is for Project and repository based on UAC workspace which returns me resourcesIds.

Reply:
Correct, but I don't understand how _both_ paths setFuzzyOwnerPredicateBasedOnUACWorkspace and setFuzzySearchPredicateBasedOnMDBOwner are new, since they're considerably bigger than any other code being deleted in this PR, given that this predicate construction existed before. Are we forgetting to delete some code?